tempesta_fw/http.c

```diff
@@ -2507,6 +2507,15 @@ tfw_http_resp_process(TfwConn *conn, struct sk_buff *skb, unsigned int off)
 				TFW_WARN("Insufficient memory "
 					 "to create a response sibling\n");
 				TFW_INC_STAT_BH(serv.msgs_otherr);
+				
+				/* 
+				 * We failed to create the sibling message,
+				 * but despite this, we have a correctly
+				 * parsed response. So we can send this
+				 * response to client before closing
+				 * server connection.
+				 */
```

Comment:
I would avoid using the word "we" in comments, and use an impersonalized style. Also, try to be concise.
_Unable to create a sibling message. Send the parsed response to the client and close the server connection._

Reply:
Corrected.